lighthouse-cli/README.md

```diff
@@ -0,0 +1,8 @@
+One time setup:
+
+`npm i -g typescript@2.0`
```

Comment:
so we can actually now just use npm packages for all of the typings.. I might migrate us to that